docker/sysbox/platform/BUILD

```diff
@@ -0,0 +1,9 @@
+package(default_visibility = ["//visibility:public"])
+
+platform(
+    name = "platform",
+    exec_properties = {
```

Comment:
If you're creating a platform then I'd rather use that then exec_properties on a per target level. Either or unless there is a specific need that I am missing

Reply:
removed the platform